pkg/util/clusterfiles/cluster_files.go

```diff
@@ -13,7 +13,7 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 */
-package cluster_files
+package clusterfiles
```

Comment:
How about shorten it to just _files_?

Reply:
I think also clusterfiles would be ok, not as long as "kubeproxyiptables" ':D this starts to sound like german